### DISCUSSION TITLE
I can not use copilot chat

### DISCUSSION THREAD

**Discussion by somgent** [community/community#57025] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi,

I have activated my copilot extension in Visual Studio Code, but I can not see the chat option in the UI. Any suggestions?
![image](https://github.com/community/community/assets/36660949/479eb8e2-762e-40bb-8a7e-56b6d21c73b5)

-Soumen

**Comment by muneebakhter** [comment]
Use VS Code Insiders

**Comment by somgent** [comment]
Thanks for your suggestion @muneebakhter . It didn't work.
![image](https://github.com/community/community/assets/36660949/51f3e153-343e-4313-99bf-2221a9a10ad4)

I can see there are couple of Github Copilot extensions. Which one works?

![image](https://github.com/community/community/assets/36660949/ac8eb8c9-4c93-4eb2-976d-190bece337e7)


Thyanks,
-Soumen

**Reply by danielalyoshin** [reply]
I would try installing both the "GitHub Copilot" and the "GitHub Copilot Chat" extensions. 

Let me know if this helps!

**Comment by somgent** [comment]
Thanks, @danielalyoshin. I have installed both extensions, but the issue remains the same. I can't see the chat option.

I also tried it on my MacBook, but it didn't work either. I'm wondering if there's any issue with my GitHub account. I purchased the service just few days ago, and the first month is supposed to be a trial version. Could that be the reason why I'm unable to access the chat feature? Additionally, in the editor, when I press the tab key, the intellisense works.
![image](https://github.com/community/community/assets/36660949/2a8fa053-1943-419a-a898-65b871a738df)

**Reply by danielalyoshin** [reply, marked as answer]
As stated [here](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat), there are a few requirements that have to be met before using GitHub Copilot Chat:

> 1. Only works with [Visual Studio Code Insiders](https://code.visualstudio.com/insiders/).
> 2. [GitHub Copilot Nightly](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly) must be installed alongside Copilot Chat.
> 3. Your GitHub account that is signed into VS Code must have an active [GitHub Copilot subscription](https://docs.github.com/en/copilot/quickstart#signing-up-for-github-copilot) and must be accepted to the GitHub Copilot Chat preview (join the [waitlist](https://github.com/github-copilot/chat_waitlist_signup/join)).

Check to make sure that you have all of the criteria met. I hope this helps!

**Comment by somgent** [comment]
@danielalyoshin , I have uninstalled all the extensions and re-installed "GitHub Copilot Nightly" followed by "GitHub Copilot Chat". However, the chat options didn't show up in the editor until I join the "waitlist" in the step 3. It took few hours for GitHub to approve (I guess).

Thanks for sharing the steps. It works now. The steps are really helpful. I appreciate for your support.

-Soumen

**Reply by danielalyoshin** [reply]
Anytime, glad I could be of help!

**Reply by KarthikPazha** [reply]
it is Intelliji Idea 2024 community edition, it was working since Yesterday.Am using Intelliji , How can i enable Github copilot chat as a part of my subscription ? After the installation of Github copilot, when i try to install Github copilot chat it keep on says , Initialing checking Github Copilot chat Prerequisites , how to resolve the issue ? generally what are the Prerequisites for Github Copilot chat ? it was working since yesterday not sure what happened today ??? please help to resolve , my authentication is successful.
